src/main/java/com/opencart/pages/SuccessSearchProductResultPage.java

```diff
@@ -0,0 +1,17 @@
+package com.opencart.pages;
+
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+/*
+ * Class that is useful in SearchProductTest
+ */
+
+public class SuccessSearchProductResultPage extends BasePage {
```

Comment:
Please, rename this page to SearchProductResultPage. The difference between successful and unsuccessful searches is only the list of products that correspond to the search phrase. In case, when search unsuccessful, that list will be empty. And message - "There is no product that matches the search criteria." will appear. So it is unnecessary to create two pages (Success/Unsuccess) Search there.  You will have only one.